compiler warnings in usbdevice/ledgerapdu.cpp

**Describe the bug**

Compiler warnings pop up from `usbdevice/ledgerapdu.cpp`.

```
usbdevice/ledgerapdu.cpp:88:16: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
        apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_FINALIZE_FULL, 0x00, 0x00));
               ^
usbdevice/ledgerapdu.cpp:88:16: note: remove std::move call here
        apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_FINALIZE_FULL, 0x00, 0x00));
               ^~~~~~~~~~                                                     ~
usbdevice/ledgerapdu.cpp:134:16: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
        apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00));
               ^
usbdevice/ledgerapdu.cpp:134:16: note: remove std::move call here
        apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00));
               ^~~~~~~~~~                                             ~
usbdevice/ledgerapdu.cpp:146:14: warning: moving a temporary object prevents copy elision [-Wpessimizing-move]
      apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00));
             ^
usbdevice/ledgerapdu.cpp:146:14: note: remove std::move call here
      apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00));
             ^~~~~~~~~~
```

**To Reproduce**
Do a clean build using `clang`.

**Expected behavior**
No compiler warnings should pop up during the build.

**Environment**

```
$ clang --version
Apple LLVM version 10.0.0 (clang-1000.11.45.5)
Target: x86_64-apple-darwin18.2.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```
`master` @ b5a6c5f4481ffb954d0ef1c7ccb1de489f783840

--- a/src/esperanza/walletextension.h
+++ b/src/esperanza/walletextension.h
@@ -29,9 +29,9 @@ class CWallet;
 class CWalletTx;
 class COutput;
 class CScheduler;
-class CRecipient;
 class CCoinControl;
 class CReserveKey;
+struct CRecipient;
 
 namespace esperanza {
 

--- a/src/usbdevice/ledgerapdu.cpp
+++ b/src/usbdevice/ledgerapdu.cpp
@@ -85,7 +85,7 @@ bool GetPreparePhaseAPDUs(const CTransaction &tx, const CCoinsViewCache &view,
         it += chunk_size;
 
         apdus_out.emplace_back(std::move(apdu));
-        apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_FINALIZE_FULL, 0x00, 0x00));
+        apdu = APDU(BTCHIP_INS_HASH_INPUT_FINALIZE_FULL, 0x00, 0x00);
       }
     }
 
@@ -131,7 +131,7 @@ bool GetSignPhaseAPDUs(const std::vector<uint32_t> &path,
     while (script_data_left > 0) {
       if (apdu.max_size == apdu.size) {
         apdus_out.emplace_back(std::move(apdu));
-        apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00));
+        apdu = APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00);
       }
 
       size_t space_left = apdu.max_size - apdu.size;
@@ -143,7 +143,7 @@ bool GetSignPhaseAPDUs(const std::vector<uint32_t> &path,
 
     if (apdu.max_size - apdu.size < 4) {
       apdus_out.emplace_back(std::move(apdu));
-      apdu = std::move(APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00));
+      apdu = APDU(BTCHIP_INS_HASH_INPUT_START, 0x80, 0x00);
     }
 
     apdu << txin.nSequence;

